### PR TITLE
Converting project to VS2022

### PR DESCRIPTION
**Pull request by Aingar**
Old bad, new better.
Also made all plugin projects with boost dependencies have the /dependencies folder added to external dependencies. Why weren't they already?

Plausible performance increase due to using newer compiler.

Test server is currently running VS2022 generated DLLs without any issues.

### CODE REVIEW

File: Plugins/Public/alley/alley.vcxproj [hunk 1]
```diff
@@ -84,8 +84,7 @@ copy /Y $(OutDir)$(TargetName).pdb  ..\..\..\Binaries\bin-vc14\flhook_plugins\pu
     <ClCompile>
       <Optimization>MaxSpeed</Optimization>
       <IntrinsicFunctions>true</IntrinsicFunctions>
-      <AdditionalIncludeDirectories>
-      </AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>A:\FLHookAingar\Dependencies;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```
Comment (dsyalex) [inline Plugins/Public/alley/alley.vcxproj:87]: ditto

File: Plugins/Public/base_plugin/Base Plugin.vc14.vcxproj [hunk 2]
```diff
@@ -114,6 +114,7 @@ copy /Y $(OutDir)$(TargetName).pdb  ..\..\..\Binaries\bin-vc14\flhook_plugins\pu
       <PrecompiledHeader />
       <WarningLevel>Level3</WarningLevel>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
+      <AdditionalIncludeDirectories>A:\FLHookAingar\Dependencies;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```
Comment (dsyalex) [inline Plugins/Public/base_plugin/Base Plugin.vc14.vcxproj:117]: ditto

File: Plugins/Public/playercntl_plugin/Player Control Plugin.vc14.vcxproj [hunk 3]
```diff
@@ -111,6 +111,7 @@ copy /Y $(OutDir)$(TargetName).pdb  ..\..\..\Binaries\bin-vc14\flhook_plugins\pu
       <AssemblerOutput>AssemblyCode</AssemblerOutput>
       <WarningLevel>Level3</WarningLevel>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
+      <AdditionalIncludeDirectories>A:\FLHookAingar\Dependencies;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```
Comment (dsyalex) [inline Plugins/Public/playercntl_plugin/Player Control Plugin.vc14.vcxproj:114]: ditto

File: Plugins/Public/JSONBuddy/JSONBuddy.vcxproj [hunk 4]
```diff
@@ -85,8 +85,7 @@ copy /Y $(OutDir)$(TargetName).pdb  ..\..\..\Binaries\bin-vc14\flhook_plugins\pu
     <ClCompile>
       <Optimization>MaxSpeed</Optimization>
       <IntrinsicFunctions>true</IntrinsicFunctions>
-      <AdditionalIncludeDirectories>
-      </AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>A:\FLHookAingar\Dependencies;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```
[the remark this answers is not in the document]
Comment (Aingar) [reply Plugins/Public/JSONBuddy/JSONBuddy.vcxproj:88]: Whoops. Also that's one of my main hard drives, I'm a rebel.

**Review by dsyalex — CHANGES_REQUESTED** [review]
needs fixing to work for people who don't write their FLHook to A: drive and aren't called Aingar :P

**Comment by dsyalex** [comment]
Cf private PR 1

**Pull request merged**